docs/tutorials/bootloader.md

```diff
@@ -154,3 +154,54 @@ After exporting a project with the `target.bootloader_img` setting, you are resp
 
 For an unmanaged bootloader:
 After exporting a project with the `target.mbed_app_start` setting, you are responsible for ensuring that a boot loader is present, if needed. Without flashing this boot loader image, the device will not boot correctly.
+
+### Prebuilt bootloaders
+
+There is an option to merge prebuilt bootloaders to your application. `mbed-os\features\FEATURE_BOOTLOADER\mbed_lib.json` lists all available targets with the corresponding prebuilt bootloader. The prebuilt bootloaders are located in the same directory.
+
+Below explains how to add the feature BOOTLOADER into your application.
+- add BOOTLOADER feature into `mbed_app.json` located in the root application directory
```

Comment:
I would change this section (lines 163 to 170) to:

```
1. Add the BOOTLOADER feature into `mbed_app.json` located in the root application directory
    ```
    "target_overrides": {
        "*": {
            "target.features_add": ["BOOTLOADER"]
        }
    }
    ```
1. Add `-C 'target.features_add=["BOOTLOADER"]'` to your `mbed compile` command line arguments
```